### PR TITLE
axios successful for one category call

### PR DESCRIPTION
**Pull request by ngray122**

### CODE REVIEW

**Comment by ngray122** [comment]
@jaygraygray Can you review?  I am able to dynamically render APi endpoint information in my Category component although I do not know if it is the correct way to go about it.

File: client/src/App.jsx [hunk 1]
```diff
@@ -2,12 +2,21 @@ import logo from "./logo.svg";
...
 import NavBarComponent from "./components/nav/NavBarComponent";
+import { Routes, Route } from "react-router";
+import Category from "./components/card/Category";
 
 function App() {
   return (
     <div className="App">
-      <NavBarComponent />
-      <Landing />
+      <>
```
Comment (jaygraygray) [inline client/src/App.jsx:11]: I don't think you need the fragment here

File: client/src/components/card/Category.jsx [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+        console.log("error!! ====> ", err);
+      });
+  }, [categoryList]);
   return (
     <div>
-      {apiList.map((item, i) => {
-        return <h1>{item.name}</h1>;
-      })}
-      <h1>Hello</h1>
+      {/* <h1>Character</h1> */}
```
Comment (jaygraygray) [inline client/src/components/card/Category.jsx:23]: Delete commented code

File: client/src/components/card/Category.jsx [hunk 3]
```diff
@@ -1,28 +1,37 @@
...
-  // }, [setApiList]);
+  useEffect(() => {
+    axios
+      // characters will be dynamic
+      .get(`https://bobsburgers-api.herokuapp.com/${header}/`)
+      .then((response) => {
+        setCategoryList(response.data);
+      })
+      .catch((err) => {
+        console.log("error!! ====> ", err);
```
Comment (jaygraygray) [inline client/src/components/card/Category.jsx:18]: Not urgent right now but you should start handling error cases in the UI instead of console logging.

Comment (ngray122) [reply client/src/components/card/Category.jsx:17]: @jaygraygray like alert()??

**Pull request merged**